hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.module.scss

```diff
@@ -0,0 +1,53 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.Cart {
+    display: flex;
+    color: color(backgroundC);
```

Comment:
This should be `background-color: color(backgroundC);`. You can remove the other `background-color: color(backgroundC);` classes below